src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -45,24 +43,8 @@ private class SearchResult : INavigateToSearchResult
                 NameMatchSpans = nameMatchSpans;
                 SecondarySort = ConstructSecondarySortString(document, declaredSymbolInfo);
 
-                var declaredNavigableItem = navigableItem as NavigableItemFactory.DeclaredSymbolNavigableItem;
-                Debug.Assert(declaredNavigableItem != null);
-
-                _lazySummary = new Lazy<string>(() => declaredNavigableItem.SymbolOpt?.GetDocumentationComment()?.SummaryText);
-                _lazyAdditionalInfo = new Lazy<string>(() =>
-                {
-                    switch (declaredSymbolInfo.Kind)
-                    {
-                        case DeclaredSymbolInfoKind.Class:
-                        case DeclaredSymbolInfoKind.Enum:
-                        case DeclaredSymbolInfoKind.Interface:
-                        case DeclaredSymbolInfoKind.Module:
-                        case DeclaredSymbolInfoKind.Struct:
-                            return FeaturesResources.project_space + document.Project.Name;
-                        default:
-                            return FeaturesResources.type_space + declaredSymbolInfo.ContainerDisplayName;
-                    }
-                });
+                Summary = summary;
+                AdditionalInformation = additionalInfo;
```

Comment:
these values are no longer lazy, now that we can compute what we need in the OOP process and just marshal it over.